### PR TITLE
NEXUS-38130 - Add resource specifications with minimum Nexus requirements

### PR DESCRIPTION
**Pull request by mpiggott**

### CODE REVIEW

File: nxrm-aws-ha-helm/values.yaml [hunk 1]
```diff
@@ -23,7 +23,15 @@ statefulset:
   container:
     image:
       repository: sonatype/nexus3
-      tag: 3.45.1
+      tag: 3.50.0
+    resources:
+      # See help documentation, these are minimum sysem requirements
```
Comment (mpiggott) [inline nxrm-aws-ha-helm/values.yaml:28]: I chose these values based on our minimum requirements in https://help.sonatype.com/repomanager3/product-information/system-requirements

**Comment by bobotimi** [comment]
We should probably add some fraction of cpu and memory to the logging containers as well. They don't need to be no where near the one for the nexus repo container

**Review by userSeprid — APPROVED** [review]

**Review by oleksiirudyk — APPROVED** [review]

File: sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml [hunk 2]
```diff
@@ -55,6 +55,13 @@ spec:
...
+          resources:
+            limits:
+              cpu: "0.2"
+              memory: "512Mi"
+            requests:
+              cpu: "0.1"
+              memory: "256Mi"
       terminationGracePeriodSeconds: 20
       containers:
       - name: nxrm-app
```
Comment (bobotimi) [inline sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml:67]: Just missing for this container.

**Review by bobotimi — APPROVED** [review]
Just one container left, otherwise looks good

File: sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml (hunk 2 above)
Comment (mpiggott) [reply sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml:67]: Added

File: sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml [hunk 3]
```diff
@@ -94,7 +109,7 @@ spec:
             secretKeyRef:
               name: nxrm-secret
               key: nxrm-admin-user
-   - name: INSTALL4J_ADD_VM_PARAMS
+        - name: INSTALL4J_ADD_VM_PARAMS
```
Comment (mpiggott) [inline sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml:112]: Indentation was off

**Pull request merged**